src/plugin.js

```diff
@@ -225,15 +225,8 @@ function doZoom(chart, percentZoomX, percentZoomY, focalPoint, whichAxes, animat
       }
     });
 
-    if (animationDuration) {
-      // needs to create specific animation mode
-      if (!chart.options.animation.zoom) {
-        chart.options.animation.zoom = {
-          duration: animationDuration,
-          easing: 'easeOutQuad',
-        };
-      }
-      chart.update('zoom');
+    if (transitionMode && chart.options.transitions[transitionMode]) {
```

Comment:
I think it could be just `chart.update('zoom')`. Not sure where we'd need to use another transition mode for it?

Reply:
Yeah, I had left the door open to more customization in other zoom modes, but maybe unnecessary. I can just leave a `useTransition` parameter in the `doZoom` function:

```js
if (useTransition) {
  chart.update('zoom');
}
```

?

Reply:
I don't think we need to check if the transition is configured, it will fall back to default animations if not. And we need to update in either case. Also feels like the `useTransition` is not needed, because one can configure the zoom transition to have no duration.

Reply:
Ok I agree for the `chart.options.transitions.zoom` check. But we need a `useTransition` parameter because there are some calls to `doZoom` where we don't want any animation. For instance when zooming with mouse scroll, using an animation creates a "latency" which is very annoying.

Reply:
I updated the PR

Reply:
Good point, sheds some light to your thinking of possibly needing separate transitions too.

It could be ideal if we could provide some additional info in the context of the transition:

```js
options: {
  transitions: {
    zoom: {
      animation: {
        duration: ctx => ctx.mode === 'scroll' || ctx.mode === 'reset' : 0 ? 500
      }
    }
  }
}
```

But as we can't we need to separate by transition.
So eventually we might have `zoomScroll`, `zoomDrag`, `zoomReset`, `pan` transitions?

Back to this PR, I think `useTransition` is good enough for this bug fix.